mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxExtensions.kt

```diff
@@ -54,6 +55,53 @@ inline fun <T, reified VM : BaseMvRxViewModel<S>, reified S : MvRxState> T.activ
         .apply { subscribe(this@activityViewModel, subscriber = { postInvalidate() }) }
 }
 
+/**
+ * Gets or creates a Fragment-scoped ViewModel scoped to this View. You will get the same instance every time for this Fragment, even
+ * through rotation, or other configuration changes.
+ *
+ * This will walk up the view hierarchy until it finds a View that is the root view for a Fragment.
+ *
+ * If the ViewModel has additional dependencies, implement [MvRxViewModelFactory] in its companion object.
+ * You will be given the initial state as well as a FragmentActivity with which you can access other dependencies to
+ * pass to the ViewModel's constructor.
+ *
+ * MvRx will also handle persistence across process restarts. Refer to [PersistState] for more info.
+ *
+ * Use [keyFactory] if you have multiple ViewModels of the same class in the same scope.
+ */
+inline fun <T, reified VM : BaseMvRxViewModel<S>, reified S : MvRxState> T.fragmentViewModel(
```

Comment:
So there is no concept of `existing` view model for views? If the fragment, or another view in the fragment, also uses this then it will grab the existing fragment view model?

I think it's confusing to have `activityViewModel` and `existinViewModel` when we don't make that distinction here.

This feels related to the discussion I brought on slack for whether `existingViewModel` is necessary